generators/app/templates/e2e.spec.js.ejs

```diff
@@ -11,5 +12,8 @@ module.exports = E2eHelpers.createE2eTest(client => {
   client.end();
 });
 
+// disable E2E test in CI if app is not in prod
 module.exports['@disabled'] =
-  !manifest.production || __BUILDTYPE__ !== 'production';
+  !registry.find(entry => entry.entryName === manifest.entryName) &&
```

Comment:
The `!registry.find()` will only be true if the application doesn't exist in the `registry`, right? I think this still needs to check to see if it's enabled in the environment.
```suggestion
  registry.find(entry => entry.entryName === manifest.entryName).template.vagovprod !== false &&
```

Entries in the `registry.json` look like:
```
  {
    "appName": "COVID-19 screener",
    "entryName": "covid19screen",
    "rootUrl":"/covid19screen",
    "template":{
      "title": "COVID-19 screening tool",
      "layout": "page-react.html",
      "vagovprod": true
    }
  },
```
so we need that `.template.vagovprod` to make sure that the application can be found in the production build. We also need it to `!== false` because if it's just not there, it also means it's in production by default (which isn't awesome, admittedly).

Reply:
yeah you're right, except we are trying to check if it is NOT in production, right? so it should be something like:
```js
registry.find(entry => entry.entryName === manifest.entryName).template.vagovprod !== true`
```

Reply:
Trouble is if `vagovprod` is just not specified—and therefore `undefined`—it'll act like it's `true`.